<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>1151</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 222, 227);
</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="mail_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>180</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>   Mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="nom_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>  Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>    ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmerajout">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>550</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="annulerajout">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>550</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Annnuler</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>180</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="adresse_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>  Adresse </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="prenom_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>  Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="programme_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Programme</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="programme">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>120</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Création invités</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateAjout">
      <property name="geometry">
       <rect>
        <x>709</x>
        <y>424</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>530</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Recherche:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>941</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Affichage invités</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modif">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>520</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Tri par:</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID desc</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NOM desc</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NOM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PRENOM desc</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="ajoutaff">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>520</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sup">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>450</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="capture">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>450</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Capture</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>120</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Calendrier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>430</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="annulermodif">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>430</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mettre à jour des invités</string>
      </property>
     </widget>
     <widget class="QLabel" name="nom_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>  Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>    ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>290</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="programme_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>230</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Programme</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="prenom_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>  Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>290</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="programme_4">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>230</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="adresse_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>  Adresse </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>230</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="mail_4">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>290</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>   Mail</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>350</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Suppression des invités</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idsupp">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>330</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>220</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>    ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmersup">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>330</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1219</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
